modules/Reports/reporting_criteria_manage.php

```diff
@@ -188,6 +188,10 @@
 
     $table->addColumn('name', __('Criteria'));
     $table->addColumn('criteriaType', __('Type'));
+    $table->addColumn('category', __('Category'))
+        ->format(function ($row) {
+            return $row['category'] ?? '';
+        });
```

Comment:
Rather than adding a new column which could take up more horizontal space, especially if category is not used by a school, lets use a formatDetails to combine this with the Type column. What do you think?

```suggestion
    $table->addColumn('criteriaType', __('Type'))
        ->description(__('Category'))
        ->formatDetails(function ($row) {
            return $row['category'] ?? '';
        });
```